.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Asscociated Issue
+ 
+Replace this text with the Jira issue resolved (e.g. CASSANDRA-####) or text specifying why this is a trivial patch.
+
+## Author(s) and Reviewers
+
+Patch by &lt;author>; reviewed by <reviewer[s]>
+
+Co-authored-by: First Last <author1@domain.com><br/>
+Co-authored-by: First2 Last2 <author2@domain.com>
+
+
+## What the patch does
+
+Replace this text with a detailed commit paragraph of why and how the change was made and relevant context.
+
+##  Recommendations:
+
+if not a trivial change:
+  - Jira ticket contains a description of: what is fixed, why it is needed, and what branches to apply it to.
+  - Tests are included.
+  - Documentation changes and/or updates are included.
+  
+
+## Notice
+By submitting this pull request, I acknowledge that I am making a contribution to the Apache Software Foundation under the terms and conditions of the Contributor's Agreement.
+
+###References:
+* The [Apache Cassandra "Contributing to Cassandra" guide](https://cassandra.apache.org/_/development/index.html) 
+* The [Apache Cassandra "Working on Documentation" guide](https://cassandra.apache.org/_/development/documentation.html)
```

Comment:
remove this, it's already part of the previous link.

Reply:
Disagree, it's duplicating information. The documentation page is embedded in the first link.

Reply:
`duplicating information` <- duplication isn't always bad. The goal here is to provide context and framing for folks to get situated, and so long as we don't literally have redundancy on the front-facing store of information here, I think we should keep it.

Reply:
but why the documentation, instead of duplicating any of the other parts of that page? (i would have thought code style was far more valuable to the newcomer).

the very first thing on the index.html page is the list of sections, documentation being one of them.